spec/support/acceptance/tar.rb

```diff
@@ -0,0 +1,39 @@
+begin
+  require 'serverspec'
+rescue LoadError
+  # Not using acceptance tests
+else
+  module Serverspec
+    module Type
+      class Tar < Command
+        def contents
+          command_result.stdout.split("\n").collect do |line|
+            line.split(' ')[5]
```

Comment:
If you use `tar -tf` (as opposed to `tar -tvf`) then it doesn't list the other columns, just the filenames. That means it also deals well with spaces in filenames.